pkg/cli/aws/aws.go

```diff
@@ -208,12 +239,88 @@ func refreshCredsViaOktaAWSCLI(ctx context.Context, copts *CredentialOptions, ac
 	return nil
 }
 
-// runCmd is a wrapper for running a command via exec.CommandContext
-// and passing through stdin/stdout/stderr.
-func runCmd(ctx context.Context, name string, args ...string) error {
-	cmd := exec.CommandContext(ctx, name, args...)
+// oktaAwsCLICmd is a convenience function that creates an exec.Cmd
+// instance for okta-aws-cli.
```

Comment:
```suggestion
// instance for the okta-aws-cli.
```

Reply:
I disagree with this suggestion, it sounds awkward putting a definite article in front of a command name.